script.js

```diff
@@ -360,6 +372,40 @@ function initMap() {
   });
 }
 
+function initLanguageSelect(selected) {
+  var fragment = document.createDocumentFragment();
+
+  Object.keys(languageMap).forEach(function(language) {
+      var opt = document.createElement('option');
```

Comment:
Nit: 2 space indent.

Also feel free to make this
`for (let language of Object.keys(languageMap))`